ci/docker/conda-python-kartothek.dockerfile

```diff
@@ -32,7 +32,14 @@ RUN conda install -c conda-forge -q \
         storefact \
         toolz \
         urlquote \
-        zstandard && \
+        zstandard \
+        attrs \
```

Comment:
Can we keep those deps in alphabetical order?

Reply:
Updated